### DISCUSSION TITLE
v0.6.0 in a docker container with Oauth

### DISCUSSION THREAD

**Discussion by jbollineni** [librespot-org/librespot#1406] [General]
Has anyone tried running librespot v0.6.0 in a container with Oauth? The `-j` flag starts a Oauth server, but on a loopback address.  So, the redirect after authentication is not working. 

Ex: 
```
[2024-11-26T18:19:22Z INFO  librespot_oauth] OAuth server listening on 127.0.0.1:5588
```

**Comment by kingosticks** [comment]
There's a section in the wiki regarding headless oauth for situations like this, use that.
